recipe/build.sh

```diff
@@ -9,11 +9,30 @@ if [[ "${target_platform}" == osx-* ]]; then
   export CXXFLAGS="${CXXFLAGS} -DTARGET_OS_OSX=1"
 fi
 
+# For builds that are cross-compiled (aarch64), we need to build a bootstrap system first.
+# https://www.erlang.org/doc/installation_guide/install-cross#Build-and-Install-Procedure_Building-With-configuremake-Directly_Building-a-Bootstrap-System
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" -eq 1 ]]; then
+
+  BOOTSTRAP_PREFIX="${PREFIX}/../bootstrap"
+  CFLAGS= LDFLAGS= ./configure \
```

Comment:
If `./configure` is not working for us, maybe we can look at [`otp_build`]( https://www.erlang.org/doc/system/install-cross#building-with-the-otp_build-script ). It will handle building both the bootstrap and for the target machine. That might save us some fiddling in combining the two ourselves

Presumably this could then be used for all builds (whether or not cross-compilation is used)

Reply:
Makes sense, I'll try that.